00-HTML-CSS-basics/vi.html

```diff
@@ -1,92 +1,50 @@
-        <label>Last name: </label><br>
-        <input name="lastName" placeholder="Required" required type="text" /><br>
-        
-        <label>Email: </label><br>
-        <input name="email" placeholder="Enter a valid email address" type="email" /><br>
-        
-        <label>Birthday: </label><br>
-        <input name="birthday" required type="date" /><br>
-        
-        <label>Choose your favourite sport: </label><br>
-            <select name="sports">
-                <option>--------</option>
-                <option>Football</option>
-                <option>Tennis</option>
-                <option>Golf</option>
-                <option>Volleyball</option>
-            </select><br><br>
-        
-        <label>Small bio: </label><br>
-        <textarea cols="50" rows="4"></textarea><br><br>
-        
-        <table>
-            <tr>
-            <th><input class="but" name="submit" type="submit" value="Submit" /></th>
-            <th><input class="but" name="reset" type="reset" value="Reset" /></th>
-            </tr>
-        </table>
-    </form>
-</div>
-
-</body>
-</html>
+    <head>
+        <meta charset="utf-8">
+        <meta content="Agustina Vighetti" name="author">
+        <meta content="Globant Bootcamp HTML and CSS Topic" name="description">
+        <meta content="width=device-width,initial-scale=1" name="viewport">
+        <link href="normalize.css" rel="stylesheet" type="text/css">
+        <link href="style_vi.css" rel="stylesheet" type="text/css">
+        <title>Form</title>
+    </head>
+    <body>
+        <div>
+            <h1>Form</h1>
+            <!-- ACTION MISSING -->
+            <form action="" method="get">
+                <label>First name: </label><br>
+                <input name="firstName" placeholder="Required" required type="text" /><br>
+                
+                <label>Last name: </label><br>
+                <input name="lastName" placeholder="Required" required type="text" /><br>
+                
+                <label>Email: </label><br>
+                <input name="email" placeholder="Enter a valid email address" type="email" /><br>
+                
+                <label>Birthday: </label><br>
+                <input name="birthday" required type="date" /><br>
+                
+                <label>Choose your favourite sport: </label><br>
+                    <select name="sports">
+                        <option>--------</option>
+                        <option>Football</option>
+                        <option>Tennis</option>
+                        <option>Golf</option>
+                        <option>Volleyball</option>
+                    </select><br><br>
+                
+                <label>Small bio: </label><br>
+                <textarea cols="50" rows="4"></textarea><br><br>
+                
+                <table>
+                    <tr>
+                    <th><input class="but" name="submit" type="submit" value="Submit" /></th>
```

Comment:
Do not use short names for anything. Also you should've been using `<button>` instead of `<input>`. HTML5 things.